src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar" data-qa="big">
+      <form
+        action="#"
+        method="get"
+        class="search-bar__form"
+      >
+        <div class="search-bar__container">
+          <div class="search-bar__content">
```

Comment:
Do you really need two div containers here?
Also, it's better to use label instead of div